tabs/receiver.js

```diff
@@ -76,6 +76,28 @@ TABS.receiver.initialize = function (callback) {
         // translate to user-selected language
         localize();
 
+        // woga65: if craft is variable pitch, default to 
+        // collective and gyro gain rather than AUX channels
+        const rcMapElement = document.getElementById('rcmap_element');
+        if (
+            RC_MAP.length === 8 ||
+            TARGET.isVariablePitch  // woga65: RC_MAP.length might be 4 if target is SITL, so make sure, heli mode is viable in simulator
+        ) {
+            rcMapElement.innerHTML = MIXER_CONFIG.platformType == PLATFORM_HELICOPTER
+                ? ` <option value="AECR12TG">Default</option>
+                    <option value="CAER12TG">Collective Pitch / Spektrum</option>
+                    <option value="AECR12TG">Collective Pitch / Futaba</option>`.trim()
+                : ` <option value="AETR1234">Default</option>
+                    <option value="TAER1234">JR / Spektrum / Graupner</option>
+                    <option value="AETR1234">Futaba / Hitec</option>`.trim();                    
```

Comment:
Please see my notes about **aux channels** from `receiver.html`.

I was going to suggest that **AECRTG78** would be better. But, I guess channel 5 has to be avoided due to _limitations_ with ELRS. So **AECR56TG** makes sense because of that.